src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Roboto">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <div class="nav-logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="Logo_Moyo" >
+        </a>
+      </div>
+
+      <nav class="box">
```

Comment:
it's better to make nav as separate block, so your nav element can have .nav class and then your header could have .header class